mypy_ls/plugin.py

```diff
@@ -187,10 +189,11 @@ def init(workspace: str) -> Dict[str, str]:
             configuration = eval(file.read())
     global mypyConfigFile
     mypyConfigFile = findConfigFile(workspace, "mypy.ini")
-    if (("enabled" not in configuration or configuration["enabled"])
-       and ("live_mode" not in configuration or configuration["live_mode"])):
+    if ("enabled" not in configuration or configuration["enabled"]) and (
+        "live_mode" not in configuration or configuration["live_mode"]
+    ):
         global tmpFile
-        tmpFile = tempfile.NamedTemporaryFile('w', delete=False)
+        tmpFile = tempfile.NamedTemporaryFile("w", delete=False)
```

Comment:
I didn't realize that my editor automatically blackens all Python files, I didn't mean to push these changes. I will revert them.

Reply:
Some uniform formatting would probably be good and I did a very poor job in that regard. Therefore, this is fine with me.

Reply:
Oh well I already pushed a revert. If you are OK with formatting with black I will open an issue and do it properly, with a pre-commit hook probably.

Reply:
That would be nice